clang/include/clang/CConv/CConv.h

```diff
@@ -63,12 +63,12 @@ class CConvInterface {
   bool BuildInitialConstraints();
 
   // Constraint Solving.
-  bool SolveConstraints();
+  bool SolveConstraints(bool CWildInfo = false);
```

Comment:
What is `CWildInfo` ? Can't guess at its purpose from the name

Reply:
Fixed.